### PR TITLE
feat(ReleaseUi):Display AssessmentSummary info from CLI xml in Release details page

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
* Conditions to be met in order to display the `Assessment Summary Information` in `Clearing Details` tab of `Release Details` page:
* There should be only `ONE CLI` xml in Release Attachments, and in case of multiple CLI files attached in release then there should be `ONLY ONE ACCEPTED CLI` xml in Release Attachment.
* If the above condition is not met, then `Show Assessment Summary Info` button will not be shown.
* `Assessment Summary` information should be present in CLI xml, else appropriate message will be shown.

Issue: #1577 

### How To Test?

- Create a Release, and upload a `CLI` xml attachment in Release.
- Now You'll see the button `Show Assessment Summary Info`.
- Clicking on this button should load the Assessment summary information in the UI.

### Screenshot:
**Assessment Summary info**
image

image

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>

### CODE REVIEW

**Comment by rudra-superrr** [comment]
Testing was successful.

**Review by smrutis1 — CHANGES_REQUESTED** [review]
Some minor review comments

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/css/components/_table.scss [hunk 1]
```diff
@@ -88,6 +88,10 @@ table  {
         }
 	}
 
+    td.comment-text {
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/css/components/_table.scss:91]: fixed Indentation

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -191,6 +191,14 @@
 	}
 
 	table {
+        thead.cursor {
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/css/main.scss]: fixed Indentation

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java [hunk 3]
```diff
@@ -766,6 +770,50 @@ private void loadSpdxLicenseInfo(ResourceRequest request, ResourceResponse respo
...
+        } catch (TException e) {
+            log.error("Cannot retrieve license information for attachment id " + attachmentContentId + " in release "
+                    + releaseId + ".", e);
+            response.setProperty("statusText", "Cannot retrieve license information for CLI");
+            response.setStatus(500);
+        }
+        try {
+            JsonGenerator jsonGenerator = JSON_FACTORY.createGenerator(response.getWriter());
+            jsonGenerator.writeStartObject();
+            if (CommonUtils.isNullOrEmptyMap(assessmentSummaryMap) || assessmentSummaryMap.size() < 1) {
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java]: Thanks, removed the second condition.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java [hunk 4]
```diff
@@ -766,6 +770,50 @@ private void loadSpdxLicenseInfo(ResourceRequest request, ResourceResponse respo
...
+
+        ComponentService.Iface componentClient = thriftClients.makeComponentClient();
+        LicenseInfoService.Iface licenseInfoClient = thriftClients.makeLicenseInfoClient();
+        Map<String, String> assessmentSummaryMap = new HashMap<>();
+
+        try {
+            Release release = componentClient.getReleaseById(releaseId, user);
+            List<LicenseInfoParsingResult> licenseInfoResult = licenseInfoClient.getLicenseInfoForAttachment(release,
+                    attachmentContentId, true, user);
+            assessmentSummaryMap = licenseInfoResult.get(0).getLicenseInfo().getAssessmentSummary();
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java]: added null and empty check for `licenseInfoResult`

**Comment by KoukiHama** [comment]
@akapti  @smrutis1 

I want to try this function, but I dont know how to make valid CLI xml for this function.
Can I make its xml by fossology?

**Pull request merged**